.github/workflows/codespell.yml

```diff
@@ -12,4 +12,5 @@ jobs:
       - uses: codespell-project/actions-codespell@master
         with:
           check_filenames: true
-          skip: ./.git,yarn.lock,*__mocks__/*
+          # skip git, yarn, mocks, and i18n non-English resources.
+          skip: ./.git,yarn.lock,*__mocks__/*,*i18n/fr*
```

Comment:
Is there a way to just ignore only the non-english i18n files?

Reply:
[I tried here](https://github.com/opentripplanner/otp-react-redux/blob/9770827d47c39ccc9103958790a402acc8f32788/.github/workflows/codespell.yml#L15), but the pattern `./i18n/!(en-US.yml)` isn't recognized for some reason (suggestions welcome).